x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/setup_guide/setup_guide.tsx

```diff
@@ -12,6 +12,7 @@ import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 
 import { ENTERPRISE_SEARCH_PLUGIN } from '../../../../../common/constants';
+import { DOCS_PREFIX } from '../../../app_search/routes';
```

Comment:
Although as a quick note on this - whenever @chriscressman changes our docs, we should update our Setup Guide links to link out to a single shared Enterprise Search authentication doc instead of using AS/WS specific auth docs